README.rst

```diff
@@ -154,7 +154,7 @@ Then we convert ``MyGreatProject.jucer`` to a new ``CMakeLists.txt`` file: ::
 
   $ cd <root>/MyGreatProject/
 
-  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake
+  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake --juce-modules="../JUCE/modules"
```

Comment:
Specifying `--juce-modules` is only necessary for `.jucer` projects that use JUCE modules from the Global Path set in Projucer. It will result in 
```
set(JUCE_MODULES_GLOBAL_PATH "${CMAKE_CURRENT_LIST_DIR}/../JUCE/modules")
```
being written in the `CMakeLists.txt` file, but `JUCE_MODULES_GLOBAL_PATH` won't be used anywhere if the `.jucer` project doesn't rely on the Global Path, so it might be quite confusing in that case.

I guess adding something like:
```diff
--- a/README.rst
+++ b/README.rst
@@ -158,6 +158,15 @@ Then we convert ``MyGreatProject.jucer`` to a new ``CMakeLists.txt`` file: ::

   <root>/MyGreatProject/CMakeLists.txt has been successfully generated.

+If ``MyGreatProject.jucer`` uses JUCE modules from the Global Path set in Projucer, we
+also need to pass ``--juce-modules="../JUCE/modules"`` to ``Jucer2Reprojucer`` ::
+
+  $ cd <root>/MyGreatProject/
+
+  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake --juce-modules="../JUCE/modules"
+
+  <root>/MyGreatProject/CMakeLists.txt has been successfully generated.
+
 Now we can build ``MyGreatProject`` using CMake: ::
```
would be an improvement, but it is quite verbose.

Reply:
Hmm, yeah, you could flip this around though so that it's a footnote rather than a "you must understand this first". Helps with the optic flow a bit and lowers the visual importance of the secondary information.

```suggestion
  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake --juce-modules="../JUCE/modules"

*Note that the ``--juce-modules`` parameter is only necessary if your jucer file uses modules from the Global Path set in Projucer.*
```

Reply:
I don't know how you made your nice inline diff...

Reply:
To get syntax highlighting, you can specify a "language" right after the 3 back-ticks that start the code block. I used `diff`, and you used `suggestion`, which turns the code block into this "Suggested change" widget.

Reply:
I really like the "footnote" idea. I would suggest the following then:
```diff
@@ -154,10 +154,14 @@ Then we convert ``MyGreatProject.jucer`` to a new ``CMakeLists.txt`` file: ::

   $ cd <root>/MyGreatProject/

-  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake
+  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake --juce-modules="../JUCE/modules"

   <root>/MyGreatProject/CMakeLists.txt has been successfully generated.

+*Note that the* ``--juce-modules`` *argument is only necessary if*
+``MyGreatProject.jucer`` *uses modules from the global "JUCE Modules" path set in
+Projucer.*
+
 Now we can build ``MyGreatProject`` using CMake: ::

   $ cd <root>/MyGreatProject/
```

(unfortunately, `*` and <tt>``</tt> cannot be nested in reST)